packages/rootage/components/radiomark.yaml

```
@@ -0,0 +1,78 @@
+        icon:
+          color: $color.palette.gray-300
+
+    size=medium:
+      enabled:
+        root:
+          size: $dimension.x5
+        icon:
+          size: $dimension.x2
+      disabled:
+        icon:
+          size: $dimension.x2_5 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)